### DISCUSSION TITLE
tokio::io::AsyncRead

### DISCUSSION THREAD

**Discussion by FallingSnow** [webrtc-rs/webrtc#39] [General]
Why is `tokio::io::AsyncRead` not implemented for `srtp::stream::Stream`?

Is it because it would require the user to read and unmarshal themselves (and prevent them from trying to unmarshal rtcp on a rtp stream)?

**Comment by metaclips** [comment]
Hello @FallingSnow we are currently deprecating Read/Write for Bytes crate. More here #14
